app/models/submission_collector.rb

```diff
@@ -187,7 +187,7 @@ def collect_next_submission
   #Use the database to communicate to the child to stop, and restart itself
   #and manually collect the submission
   #The third parameter enables or disables the forking.
-  def manually_collect_submission(grouping, rev_num, async=true)
+  def manually_collect_submission(grouping, rev_num, apply_late_penalty, async=true)
```

Comment:
Line is too long. [84/80]<br>Surrounding space missing in default value assignment.